tools/setup-buildtools.cmd

```diff
@@ -52,7 +56,6 @@ if ERRORLEVEL 1 (
 )
 
 REM Install it
-vcpkg integrate install
```

Comment:
Re. this line - I honestly do not remember what was the merit of having it. It does cause some harm, however, because from the bundled zlib, sqlite, google test, and the ones installed in vcpkg directory - local build would improperly pick the vcpkg-integrated libraries after this command. Thus, even if we reinstate it at some point - this line seems to be detrimental.